bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/FileOrRendition.java

```diff
@@ -312,7 +312,6 @@ public Session getSessionForHost(URI uri) throws IOException {
                     session.setPassword(clientProvider.getPassword());
                     session.connect();
                 } catch (JSchException ex) {
-                    Logger.getLogger(FileOrRendition.class.getName()).log(Level.SEVERE, null, ex);
```

Comment:
We should log this properly too, I think.  Errors in this class don't always bubble up where they can be inspected easily.